pkg/updates/updates.go

```diff
@@ -0,0 +1,79 @@
+package updates
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/go-chi/chi"
+	"github.com/google/uuid"
+	"github.com/redhatinsights/edge-api/pkg/devices"
+)
+
+// MakeRouter adds support for operations on update
+func MakeRouter(sub chi.Router) {
+	sub.Get("/", getDevices)
+	sub.Route("/device_uuid={device_uuid}", func(r chi.Router) {
```

Comment:
@chambridge I'd suggest it be namespaced like this:
```suggestion
	sub.Route("/device/{device_uuid}", func(r chi.Router) {
```

Reply:
actually on second thought it probably should just be
```suggestion
	sub.Route("/", func(r chi.Router) {
```
and handle params in the context manager